src/Illuminate/Auth/Access/Gate.php

```diff
@@ -318,27 +318,27 @@ public function after(callable $callback)
     }
 
     /**
-     * Determine if the given ability should be granted for the current user.
+     * Determine if all of the given abilities should be granted for the current user.
      *
-     * @param  string  $ability
+     * @param  iterable|string  $abilities
      * @param  array|mixed  $arguments
      * @return bool
      */
-    public function allows($ability, $arguments = [])
+    public function allows($abilities, $arguments = [])
     {
-        return $this->check($ability, $arguments);
+        return $this->check($abilities, $arguments);
     }
 
     /**
-     * Determine if the given ability should be denied for the current user.
+     * Determine if all of the given abilities should be denied for the current user.
```

Comment:
Is this wording correct? Wouldn't it be denied if *any* of the given abilities is not granted?

Reply:
@taylorotwell good catch, fixed comments and added tests for several cases to be sure it works as expected.